pilz_teleoperation/src/pilz_teleoperation/teleoperation_driver.py

```diff
@@ -213,7 +208,6 @@ def _send_updated_jog(self):
         if self.__key_input_is_new_enough(self.__last_jog_msg):
             new_jog = _TeleoperationJointJog(joint_jog=self.__last_jog_msg)
             new_jog.choose_joint_to_jog(self._settings.joint)
-            new_jog.scale_linear_velocity(self._settings.linear_velocity)
```

Comment:
this method is renamed above, but is it still used somewhere?
if not, why not?

Reply:
It's used in line 182